File: components/java/DocWorkbenchWebApiJAR/src/main/java/com/infosys/ainauto/docwb/web/data/RecommendedActionData.java
/** =============================================================================================================== *
 * Copyright 2019 Infosys Ltd.                                                                                    *
 * Use of this source code is governed by Apache License Version 2.0 that can be found in the LICENSE file or at    *
 * http://www.apache.org/licenses/ 
 * ================================================================================================================ *
 */

package com.infosys.ainauto.docwb.web.data;

public class RecommendedActionData {

	private float confidencePct;
	private int actionNameCde;
	private float recommendedPct;

	public float getConfidencePct() {
		return confidencePct;
	}

	public RecommendedActionData setConfidencePct(float confidencePct) {
		this.confidencePct = confidencePct;
		return this;
	}

	public void setActionNameCde(int actionNameCde) {
		this.actionNameCde = actionNameCde;
	}

	public int getActionNameCde() {
		return actionNameCde;
	}

	public float getRecommendedPct() {
		return recommendedPct;
	}

	public void setRecommendedPct(float recommendedPct) {
		this.recommendedPct = recommendedPct;
	}
}

File: components/java/DocWorkbenchUIWAR/src/main/angular/src/app/data/recommended-action-data.ts
/** =============================================================================================================== *
 * Copyright 2018 Infosys Ltd.                                                                                    *
 * Use of this source code is governed by Apache License Version 2.0 that can be found in the LICENSE file or at    *
 * http://www.apache.org/licenses/ 
 * ================================================================================================================ *
 */


import { ActionData } from './action-data';

export class RecommendedActionData {

    constructor(
        public taskTypeCde: number,
        public recommendedPct: number,
        public confidencePct: number,
        public actionNameCde: number,
        public actionDataList: ActionData[]) { }

}
